lib/usb/usb_fx07_common.c

```diff
@@ -262,6 +262,23 @@ void stm32fx07_poll(usbd_device *usbd_dev)
 		return;
 	}
 
+	/*
+	 * There is no global interrupt flag for transmit complete.
+	 * The XFRC bit must be checked in each OTG_DIEPINT(x).
+	 */
+	for (i = 0; i < 4; i++) { /* Iterate over endpoints. */
```

Comment:
This number is too small for the OTG_HS device, which also uses this code isn't it?